### ISSUE TITLE
I want to get the minutes from OnGridClick, but im only getting the hours

### ISSUE THREAD

**Issue by Nearlz** [hoangnm/react-native-week-view#148]
Hi, hope someone here could help me,

I want to add events at the moment of clicking a block, it only works with hours like 11:00 and 12:00, but when I click on the 11:30 it keeps adding at 11:00, I know I can specify minutes while setting events, but I can't get minutes from OnGridClick(), I'm only getting flat hours. Any ideas?

![image](https://user-images.githubusercontent.com/37839457/143065729-0603f906-93cd-4d0a-821f-a0268d57d1ed.png)

![image](https://user-images.githubusercontent.com/37839457/143069010-7e072358-c80b-41c2-96a5-8c913ddb21d6.png)

**Comment by rdewolff** [comment]
Could you add a bit more info and code on how you are doing this ? Ideally setup a code sandbox with your project so we can help u faster 🤞

**Comment by Nearlz** [comment]
We want to do a reservation app, but it is for half an hour, like if someone make a reservation for 11:30 it will end at 12:00, and the event should be added when pressing the gap between those two hours on the screen. It works fine if the reservation starts at 11:00, but we can't get the minutes from the OnGlidClick() for a reservation at 11:30, even when i touch it.

The timeStep is set on 30 for that purpose

In short, i want to add a event on the red square but it shows on the previous block.

![image](https://user-images.githubusercontent.com/37839457/143076041-52805184-0df9-4c84-ac30-f9cbb33c6baa.png)

**Comment by pdpino** [comment]
The `onGridClick` callback has precision only up to the hours (ignores minutes and seconds), since finger pressing is usually not so precise. So @Nearlz what you are seeing is expected, precision is only up to 11:00 or 12:00, sadly.

I think we can add precision up to minutes and seconds in the `onGridClick()` callback, but warn the developer that depending on how the agenda looks on the screen, this may not be so precise.

**Comment by pdpino** [comment]
@Nearlz now you can get the minutes (and even seconds) from the `date` object: `date.getMinutes()`

In the master branch for now, will be released soon 🙂

**Comment by hoangnm** [comment]
it's been released in version 0.8.0.
